WalletWasabi.Tests/UnitTests/Clients/PreventSleepTests.cs

```diff
@@ -0,0 +1,14 @@
+using WalletWasabi.Helpers;
+using Xunit;
+
+namespace WalletWasabi.Tests.UnitTests.Clients
+{
+	public class PreventSleepTests
+	{
+		[Fact]
+		public void PreventSleep()
+		{
+			EnvironmentHelpers.KeepSystemAwake();
```

Comment:
What does this test actually?

Reply:
Checks if the function not throwing an exception on different OS-s.

Reply:
That's really not obvious without a comment. 

If I understand that correctly, the side effect of running that test is that on the developer machine, the system won't be put to sleep as expected. CI won't be affected as those need to be virtual machines.

Reply:
The fuctionality itself won't be tested - just the possibility to run the command on all CI-s. I am adding a comment.